pkg/api/types.go

```diff
@@ -477,6 +481,10 @@ type PersistentVolumeClaimSpec struct {
 	// claim. When set to non-empty value Selector is not evaluated
 	// +optional
 	VolumeName string
+	// Name of the StorageClass where the desired volume belongs to.
```

Comment:
`required` seems to be used everywhere else in the claim, hence "Name of the StorageClass required by the claim".

Good catch, btw.